website/integrations/services/apple/index.md

```diff
@@ -0,0 +1,385 @@
+>
+> -- [Apple Business Manager](https://www.apple.com/business/)
+
+:::info Apple Device Management Platforms
+
+Apple repackages their device management platform into three brands to cater to different audiences:
+
+- Apple Business Manager: Large organizations
+- Apple Business Essentials: Small businesses
+- Apple School Manager: Educational institutions
+
+While this integration guide focuses on Business Manager, the instructions are applicable to all three platforms with minor changes to the terminology.
+
+:::
+
+## Authentication Flow
+
+This sequence diagram shows a high-level flow between the user's MacBook, authentik, and Apple.
+
+```mermaid
+sequenceDiagram
+    autonumber
+    participant User
+    participant authentik
+    participant Apple
+
+    User->>Apple: User sign-in via iCloud
+    Note over Apple: 🔍 Email domain is federated
+    Apple-->>authentik: Redirect to authentik
+    Note over authentik: ✅ Authenticate
+    authentik-->>Apple: Identity verified
+    Apple-->>User: Device enrolled!
+
+```
+
+In short, Apple Business Manager recognizes the email domain
+as a federated identity provider controlled by authentik. When a user signs in with their email address, Apple redirects them to authentik for authentication. Once authenticated, Apple enrolls the user's device and grants access to Apple services.
+
+---
+
+## Preparation
+
+By the end of this integration, your users will be able to enroll their Apple devices using their authentik credentials.
+
+You'll need to have authentik instance running and accessible on an HTTPS domain, and an Apple Business Manager user with the role of Administrator or People Manager.
+
+:::warning Caveats
+
+Be aware that Apple Business Manager imposes the following restrictions on federated authentication:
+
+- Federated authentication should use the user’s email address as their username. Aliases aren’t supported.
+- Existing users with an email address in the federated domain will automatically be converted to federated authentication, effectively _taking ownership_ of the account.
+- User accounts with the role of Administrator, Site Manager, or People Manager can’t sign in using federated authentication; they can only manage the federation process.
+
+:::
+
+### Placeholders
+
+The following placeholders are used:
+
+- `authentik.company`: The FQDN of the authentik installation.
+
+## authentik configuration
+
+The workflow to configure authentik as an identity provider for Apple Business Manager involves creating scope mappings, signing keys, a Shared Signals Framework provider, and an OIDC provider. Together, these components will handle the authentication flow and backchannel communication between authentik and Apple Business Manager.
+
+### 1. Create scope mappings
+
+#### 1.1. Mapping the user's profile information
```

Comment:
```suggestion
The first step is to map the user's profile information.
```

Reply:
Ah, I think after staring at this for a wile (and it is very complex steps and sub-steps), I think the best way to avoid MS style numbering is to use a simple sentence explainingg the substeps. Let's see how this looks.